ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -330,18 +332,6 @@ object ErrorCategory {
       )
       with ErrorCategory
 
-  @Deprecated
-  object IsAbortShouldBePrecondition
```

Comment:
Removing this one as I couldn't find references to it in Daml or Canton

Reply:
:+1: